test/data/helm-chart/template.yaml

```diff
@@ -553,6 +401,275 @@ spec:
+                              importName: EntityArgoCDOverviewCard
+                              mountPoint: entity.page.overview/cards
+                            - config:
+                                if:
+                                  allOf:
+                                  - isArgocdAvailable
+                                layout:
+                                  gridColumn: 1 / -1
+                              importName: EntityArgoCDHistoryCard
+                              mountPoint: entity.page.cd/cards
+                  - disabled: false
+                    package: ./dynamic-plugins/dist/roadiehq-backstage-plugin-argo-cd-backend-dynamic
+                  - disabled: false
+                    package: ./dynamic-plugins/dist/roadiehq-scaffolder-backend-argocd-dynamic
+                  - disabled: false
+                    package: ./dynamic-plugins/dist/backstage-plugin-techdocs-backend-dynamic
+                  - disabled: false
+                    package: ./dynamic-plugins/dist/backstage-plugin-techdocs
+                  - disabled: false
+                    package: ./dynamic-plugins/dist/backstage-plugin-kubernetes
+                  - disabled: false
+                    package: ./dynamic-plugins/dist/backstage-plugin-kubernetes-backend-dynamic
+                    pluginConfig:
+                      kubernetes:
+                        clusterLocatorMethods:
+                        - clusters:
+                          - authProvider: serviceAccount
+                            name: rhdh-kubernetes-plugin
+                            serviceAccountToken: <token>
+                            skipTLSVerify: true
+                            url: https://kubernetes.default.svc
+                          type: config
+                        customResources:
+                        - apiVersion: v1
+                          group: tekton.dev
+                          plural: pipelineruns
+                        - apiVersion: v1
+                          group: tekton.dev
+                          plural: taskruns
+                        serviceLocatorMethod:
+                          type: multiTenant
+                  - disabled: false
+                    package: ./dynamic-plugins/dist/janus-idp-backstage-plugin-tekton
+                    pluginConfig:
+                      dynamicPlugins:
+                        frontend:
+                          janus-idp.backstage-plugin-tekton:
+                            mountPoints:
+                            - config:
+                                if:
+                                  allOf:
+                                  - isTektonCIAvailable
+                                layout:
+                                  gridColumn: 1 / -1
+                                  gridRowStart: 1
+                              importName: TektonCI
+                              mountPoint: entity.page.ci/cards
+              EOF
```

Comment:
Should we store these lines as a standalone template file?

Reply:
This file is just a rendering of the helm chart. It's there as a summary of the changes to the chart's behavior.

That file is not used by the chart at all.